master/pkg/fluent/fluent.go

```diff
@@ -0,0 +1,227 @@
+package fluent
```

Comment:
We might could add a least one unit tests that checks a fully saturated fluent config is formed correctly